adding_service_cluster/adding-service.adoc

```diff
@@ -14,7 +14,7 @@ endif::[]
 include::modules/adding-service-existing.adoc[leveloffset=+1]
 include::modules/access-service.adoc[leveloffset=+1]
 include::modules/deleting-service.adoc[leveloffset=+1]
-include::modules/deleting-service-cli.adoc[leveloffset=+1]
+//include::modules/deleting-service-cli.adoc[leveloffset=+1]
```

Comment:
i'm not really sure what this does (don't know the templating engine), but if that's a line-comment - should we remove this line altogether?

Reply:
@eliranmal thank you for highlighting this. I opted to comment this `include` statement out instead of removing it, so that we can easily reintroduce the section if the OCM CLI becomes generally available.